erpnext/accounts/party.py

```diff
@@ -515,6 +515,16 @@ def get_dashboard_info(party_type, party, loyalty_program=None):
 			fields=["company", "sum(grand_total) as grand_total", "sum(base_grand_total) as base_grand_total"]
 		)
 
+	company_wise_monthly_total = frappe.get_all(doctype,
```

Comment:
Check this one again, I'm seeing all the invoices posted in June instead of none in July